### DISCUSSION TITLE
dac_output, LED_BUILTIN and override pins

### DISCUSSION THREAD

**Discussion by pschatzmann** [pschatzmann/arduino-audiokit#67] [Announcements]
- Provide LED_BUILTIN for LED
- Allow to override pins in generic board_def.h when using PlatformIO
- proper support to be able to select  earphones or amplifier in dac_output

<hr /><em>This discussion was created from the release <a href='https://github.com/pschatzmann/arduino-audiokit/releases/tag/v0.6.3'>dac_output, LED_BUILTIN and override pins </a>.</em>

**Comment by kaushleshchandel** [comment]
Very useful. Thanks
